test/mockLightspeedServer/server.ts

```diff
@@ -48,6 +48,7 @@ export default class Server {
       res.send({
         access_token: "ACCESS_TOKEN",
         refresh_token: "REFRESH_TOKEN",
+        expires_in: 3600,
```

Comment:
In order to prevent calling token refresh when the mock lightspeed server is used.